README.md

```diff
@@ -16,13 +15,63 @@ And then execute:
 $ bundle
 ```
 
-Or install it yourself as:
-```bash
-$ gem install govuk_publishing_components
+### Mount the component guide
+
+Mount the component guide in your application:
+
+```ruby
+# config/routes.rb
+unless Rails.env.production?
+  mount GovukPublishingComponents::Engine, at: "/component-guide"
+end
+```
+
+For example, if your application was government-frontend, the component guide would be found at:
+`government-frontend.dev.gov.uk/component-guide`
+
+### Write components
+
+`govuk_publishing_components` expects the following components structure in your application:
+
+* a Partial View in `app/views/components` eg `_component-name.html.erb` - The template logic and markup, also defines the arguments expected
+* Documentation in `/app/views/components/docs` - a `.yml` per component, describing the component and containing fixture data.
+* a SCSS module in `app/assets/stylesheets/components` - The styling of the component
+* a Javascript module in `app/assets/javascripts/components` - JS behaviour of the component
+* a unit test in `/test/components` - testing the component renders correctly based on parameters passed in
```

Comment:
Also, even though we're not giving people a comprehensive guide on how to write a component here, it might still be worth linking to our component principles?